web-wallet/src/lib/containers/MigrateContract/MigrateContract.svelte

```diff
@@ -226,20 +284,21 @@
               height="27"
```

Comment:
Let's keep the width and the height of both of the images:

```
width="37"
height="27"
```

so that switching between the chains looks like this:
https://github.com/user-attachments/assets/172bc678-7cb9-45d5-bfe7-f237b186e96d


Otherwise we have a layout jump:
https://github.com/user-attachments/assets/f6684cce-76f4-4f5d-a8d5-68a165bccc9e